system_files/desktop/shared/usr/libexec/waydroid-container-start

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/bash
 
-source /etc/default/waydroid-launcher
-source /etc/default/steamos-nested-desktop
```

Comment:
Note: No variable from `/etc/default/steamos-nested-desktop` is being used; therefore that `source` statement was unneeded.